docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -177,96 +120,67 @@ You should see the contents of the project's `data/` directory printed to your t
 08_reporting
 ```
 
-### Create a new Databricks notebook
-
-Now that your project is available on Databricks, you can run it on a cluster using a notebook.
-
-To run the Python code from your Databricks Repo, [create a new Python notebook](https://docs.databricks.com/notebooks/notebooks-manage.html#create-a-notebook) in your workspace. Name it `iris-databricks` for traceability and attach it to your cluster:
-
-![Create a new notebook on Databricks](../../meta/images/databricks_notebook_creation.png)
-
-### Run your project
-
-Open your newly-created notebook and create **four new cells** inside it. You will fill these cells with code that runs your project. When copying the following code snippets, remember to replace `<databricks_username>` with your username on Databricks such that `project_root` correctly points to your project's location.
-
-1. Before you import and run your Python code, you'll need to install your project's dependencies on the cluster attached to your notebook. Your project has a `requirements.txt` file for this purpose. Add the following code to the first new cell to install the dependencies:
-
-```ipython
-%pip install -r "/Workspace/Repos/<databricks_username>/iris-databricks/requirements.txt"
-```
-
-2. To run your project in your notebook, you must load the Kedro IPython extension. Add the following code to the second new cell to load the IPython extension:
-
-```ipython
-%load_ext kedro.ipython
-```
-
-3. Loading the extension allows you to use the `%reload_kedro` line magic to load your Kedro project. Add the following code to the third new cell to load your Kedro project:
-
-```ipython
-%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks
+## Modify spark hook
```

Comment:
should we just update the starter/add on so this is default?

Reply:
Sounds like the right approach. I'll test both guides using a new starter along with a fresh Kedro hook to ensure everything functions as expected

Reply:
@datajoely , the integration described in hooks.py, following the guidance from the tutorial by @diegoliraQB on the Kedro blog (https://kedro.org/blog/how-to-integrate-kedro-and-databricks-connect), seems to be working smoothly using asset bundles and databricks-connect. So, it's ready to be used in the [databricks-starter](https://github.com/kedro-org/kedro-starters/blob/main/databricks-iris/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/%7B%7B%20cookiecutter.python_package%20%7D%7D/hooks.py).

I'm considering whether it makes sense for me to open a pull request (PR). However, since I didn't create this code snippet myself, I'm not entirely sure about the protocol.


This documentation pull request (PR) will likely depend on the starter, I suppose.